v1.0/sql-faqs.md

```diff
@@ -86,6 +86,36 @@ Not at this time, but storing a 16-byte array in a [`BYTES`](bytes.html) column
 
 When an [`ORDER BY`](select.html#sorting-retrieved-values) clause is not used in a `SELECT` query, retrieved rows are not sorted by any consistent criteria. Instead, CockroachDB returns them as the coordinating node receives them.
 
+## Why are my `INT` columns returned as strings in JavaScript?
+
+In CockroachDB, all `INT`s are represented with 64 bits of precision, but JavaScript numbers only have 53 bits of precision. This means that large integers stored in CockroachDB are not exactly representable as JavaScript numbers. For example, JavaScript will round the integer `235191684988928001` to the nearest representable value, `235191684988928000`. Notice that the last digit is different. This is particularly problematic when using the `unique_rowid()` [function](functions-and-operators.html), since `unique_rowid()` nearly always returns integers that require more than 53 bits of precision to represent.
+
+To avoid this loss of precision, Node's [`pg` driver](https://github.com/brianc/node-postgres) will, by default, return all CockroachDB `INT`s as strings.
+
+~~~ javascript
+// Schema: CREATE TABLE users (id INT DEFAULT unique_rowid(), name STRING);
+pgClient.query("SELECT id FROM users WHERE name = 'Roach' LIMIT 1", function(err, res) {
+  var idString = res.rows[0].id;
+  // idString == '235191684988928001'
```

Comment:
doesn't really matter since it's a comment, but my javascript head alarm goes off whenever it sees `==`, since `===` should generally be used instead.

Reply:
Done.